scripts/install_pytorch.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -xe
+
+
+TMP_DIR=tmp
+
+function _cleanup() {
+    [[ -d ${TMP_DIR} ]] && rm -rf ${TMP_DIR}
+}
+
+trap _cleanup EXIT
+
+# Adjust as per the version used in CI.
+PYTORCH_VERSION=0.4.0a0
+PYTORCH_BUILD_COMMIT=e40425f
```

Comment:
Do you think we're likely to get these commits out-of-sync by forgetting to update this file? Or are more likely to shoot ourselves in the foot being too clever via
```sh
PYTORCH_BUILD_COMMIT=$(grep 'git checkout .* a well-tested commit' README.md | cut -f3 -d' ')
```

Reply:
I like that. :) And if anyone changes the README in a way that breaks the script, we 'll catch it in CI.